mixer/adapter/stackdriver/helper/common.go

```diff
@@ -93,9 +95,14 @@ func (md *Metadata) FillProjectMetadata(in map[string]string) {
 func ToOpts(cfg *config.Params) (opts []gapiopts.ClientOption) {
 	switch cfg.Creds.(type) {
 	case *config.Params_ApiKey:
-		opts = append(opts, gapiopts.WithAPIKey(cfg.GetApiKey()))
+		if key := cfg.GetApiKey(); key != "" {
```

Comment:
Should default app credential be used as fallback?

Reply:
I thought if no option is set, the SDK will use the ADC. No?

Reply:
Oh ok. Please add a warning log in both cases so that user would know default credential would be used as fallback.

Reply:
Added some warning messages.